### DISCUSSION TITLE
Windows Package Manager 1.9.2151-preview

### DISCUSSION THREAD

**Discussion by ryfu-msft** [microsoft/winget-cli#4713] [Releases]
This is the fifth development build after the Windows Package Manager 1.8 build for Windows 10(1809+) and Windows 11.
This build will be released to Windows Insider Dev builds and Windows Package Manager Insiders.

Experimental features are enabled in this release. Run `winget features` to see which experimental features are enabled or disabled. Add the following to your settings (`winget settings`) file to enable the experimental features.

```JSON
    "experimentalFeatures": {
        "configureSelfElevate": true,
        "configureExport": true
    }
```

## What's Changed
* Update valijson to v1.0.2 by @yao-msft in https://github.com/microsoft/winget-cli/pull/4588
* Use atomic functions to ensure single initialization of tracking catalog by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4592
* Standardize line endings in expect.txt by @Trenly in https://github.com/microsoft/winget-cli/pull/4595
* Fix service connection for ESRP codesigning for utils nuget package by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4597
* Configuration apply queueing by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4590
* Ensure Agreements don't terminate in dependency flow by @Trenly in https://github.com/microsoft/winget-cli/pull/4594
* Remove libyaml Dockerfile by @florelis in https://github.com/microsoft/winget-cli/pull/4604
* Update README.md and Pull Request and Issue templates by @skanda890 in https://github.com/microsoft/winget-cli/pull/4599
* Attempt to update dependencies when not registered by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4606
* Remove experimental from index v2 by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4607
* ARM64 build and test fixes by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4608
* Use shared manifest in COM server by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4605
* Add AnyCPU to .gitignore by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4621
* Fix for attempted use of capability API before it was introduced by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4620
* Fix Component Governance by @msftrubengu in https://github.com/microsoft/winget-cli/pull/4629
* Update documentation errors by @Trenly in https://github.com/microsoft/winget-cli/pull/4649
* Update Zlib library dependency for PureLib by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4654
* update curl vcpkg version to 8.8.0 by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4655
* Pull in latest SFSClient by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4670
* Remove prerelease, update icon uri, and clean up for PowerShell modules by @ryfu-msft in https://github.com/microsoft/winget-cli/pull/4672
* Use ContinueWith rather than finally for coroutine result types by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4669
* Store configuration status and enable remote watch events by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4647
* Update README.md by @skanda890 in https://github.com/microsoft/winget-cli/pull/4687
* Update metadata collection tooling by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4688
* Skip uninstall attempt when installed package version is not present by @JohnMcPMS in https://github.com/microsoft/winget-cli/pull/4685
* Upgrade to check-spelling v0.0.22 by @jsoref in https://github.com/microsoft/winget-cli/pull/3896
* Fix downloader selection by @florelis in https://github.com/microsoft/winget-cli/pull/4696

## New Contributors
* @skanda890 made their first contribution in https://github.com/microsoft/winget-cli/pull/4599

**Full Changelog**: https://github.com/microsoft/winget-cli/compare/release-v1.9.1792-preview...release-v1.9.2151-preview

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/winget-cli/releases/tag/v1.9.2151-preview'>Windows Package Manager 1.9.2151-preview</a>.</em>

**Comment by skanda890** [comment]
How do I install the preview.

**Reply by mdanish-kh** [reply]
You can download and install the `.msixbundle` package from GitHub Releases ([download link](https://github.com/microsoft/winget-cli/releases/download/v1.9.2151-preview/Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.msixbundle)) or from https://aka.ms/getwingetpreview.

**Reply by skanda890** [reply]
Thanks!
